### PR TITLE
doc: MetadataLastModified filter

### PR DESCRIPTION
**Pull request by lucarin91**
#### :question: What

Update site for PR #133.

#### :white_check_mark: Checklists
- [x] Have the steps in [CONTRIBUTING.md](./CONTRIBUTING.md) been followed to update a Go module?

### CODE REVIEW

File: content/en/docs/Components/Filters/MetadataLastModified.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+{{% pageinfo color="primary" %}}
+
+**Read the [API documentation &raquo;](https://pkg.go.dev/github.com/AdRoll/baker/filter#MetadataLastModified)**
+{{% /pageinfo %}}
+
+## Filter _MetadataLastModified_
+
+### Overview
+
+Extract the last modified timestamp from the record Metadata and write it on a selected field.
```
Comment (tommyblue) [inline content/en/docs/Components/Filters/MetadataLastModified.md:16]:
```suggestion
Extract the "last modified" timestamp from the record Metadata and write it to the selected field.
```

File: content/en/docs/Components/Filters/MetadataLastModified.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+### Configuration
+
+Keys available in the `[filter.config]` section:
+
+| Name     |  Type  | Default | Required | Description                               |
+| -------- | :----: | :-----: | :------: | ----------------------------------------- |
+| DstField | string |         |   true   | field name into which write the timestamp |
```
Comment (tommyblue) [inline content/en/docs/Components/Filters/MetadataLastModified.md:24]:
```suggestion
| DstField | string |         |   true   | field name into which write the timestamp to |
```

File: content/en/docs/Components/Filters/MetadataLastModified.md (hunk 1 above)
Comment (tommyblue) [reply content/en/docs/Components/Filters/MetadataLastModified.md:16]: I'd maybe add some more info or links about this "last modified" timestamp. Reading this doc isn't really clear what we're talking about

**Comment by lucarin91** [comment]
I will improve the filter documentation directly in the filter source file in PR #133.

**Pull request closed without merging**